lib/status-bar-manager.coffee

```diff
@@ -0,0 +1,61 @@
+{CompositeDisposable, Disposable} = require 'atom'
```

Comment:
Would you mind converting this and the new spec file to JavaScript?  We're trying to avoid adding any new Coffeescript files.